playbooks/aws/README.md

```diff
@@ -15,19 +13,41 @@ be available soon.
 If the deployer wishes to build their own AMI for provisioning, instructions
 to do so are provided here.
 
-### Where do I start?
+### High-level overview
+- prerequisites.yml - Provision VPC, Security Groups, SSH keys, if needed.  See PREREQUISITES.md for more information.
+- build_ami.yml - Builds a custom AMI.  See BUILD_AMI.md for more information.
+- provision.yml - Create a vpc, elbs, security groups, launch config, asg's, etc.
+- install.yml - Calls the openshift-ansible installer on the newly created instances
+- provision_nodes.yml - Creates the infra and compute node scale groups
+- accept.yml - This is a playbook to accept infra and compute nodes into the cluster
+- provision_install.yml - This is a combination of all 3 of the above playbooks. (provision, install, and provision_nodes as well as accept.yml)
+
+The current expected work flow should be to provide an AMI with access to Openshift repositories.  There should be a repository specified in the `openshift_additional_repos` parameter of the inventory file. The next expectation is a minimal set of values in the `provisioning_vars.yml` file to configure the desired settings for cluster instances.  These settings are AWS specific and should be tailored to the consumer's AWS custom account settings.
+
+## How do I begin?
+
+### Ansible configuration
+The Ansible rpm shipped /etc/ansible/ansible.cfg handles aws provision and openshift-ansible plays and tasks correctly.  There is no need to modify it further.
+
+It is common to execute installer as an unpriviledged user.  That user will not have access to /etc/ansible.  The installer will cache facts locally.  To avoid permissions issues set an env variable to tell Ansible where to cache facts.
+```
+$ cd ~/git/openshift-ansible
+$ export FACT_PATH=${HOME}
```

Comment:
Unfortunately, `openshift-ansible-contrib` presumes `openshift-ansible` is installed under `/usr/share/ansible`:  https://github.com/openshift/openshift-ansible-contrib/blob/68b1ff38f693881471b6753827308063b6f5ef57/reference-architecture/aws-ansible/playbooks/roles/pre-install-check/tasks/main.yaml#L7-L10
Also, is exporting a custom FACT_PATH required?  It also differs from the caching settings in this repo's ansible.cfg.  https://github.com/openshift/openshift-ansible/blob/f622e8403ffdce31441cead5768c1aadbd90cafc/ansible.cfg#L21-L22